data/channels.csv

```diff
@@ -5906,6 +5910,7 @@ DobleCTelevision.pe,Doble C Television,Doble C Televisión,,,PE,PE-JUN,Mazamari,
 DocBox.mn,Doc Box,,,,MN,,,c/MN,mon,,FALSE,,,,,https://i.imgur.com/ziZpmv0.png
 DoceMisiones.ar,Doce Misiones,,,,AR,,,c/AR,spa,,FALSE,,,,http://www.canal12misiones.com/,https://i.imgur.com/agVTEeO.png
 Doctor.ru,Doctor,Доктор,,,RU,,,c/RU,rus,education;lifestyle,FALSE,,,,https://doc-tv.ru/,https://i.imgur.com/VNirxxn.png
+DoctorLive.in,Doctor Live,,,,IN,,,c/IN,mal,lifestyle,FALSE,,,,https://doctorlivemedia.in/,https://i.imgur.com/ZKGWABt.png

```

Comment:
```suggestion
DoctorLive.in,Doctor Live,,,,IN,,,c/IN,mal,lifestyle,FALSE,,,,https://doctorlivemedia.in/,https://i.imgur.com/D6cMaVs.png
```
Removed the transparent background around the logo.

Reply:
It already had transparency!